ghost/i18n/locales/nl/portal.json

```diff
@@ -1,148 +1,149 @@
+  "Click here to retry": "Click hier om opnieuw te proberen",
+  "Close": "Sluiten",
+  "Comments": "Reacties",
+  "Complimentary": "Gratis",
+  "Confirm": "Bevestigen",
+  "Confirm cancellation": "Bevestig opzegging",
+  "Confirm subscription": "Bevestig abonnement",
+  "Contact support": "Neem contact op met onze ondersteuning",
+  "Continue": "Doorgaan",
+  "Continue subscription": "Doorgaan met abonnement",
+  "Could not sign in. Login link expired.": "Kan niet inloggen. De inlog link is verlopen.",
+  "Could not update email! Invalid link.": "Kan je e-mail niet wijzigen! Foute link.",
+  "Create a new contact": "Maak een nieuw contact aan.",
+  "Current plan": "Huidig abonnement",
+  "Delete account": "Account verwijderen",
+  "Didn't mean to do this? Manage your preferences <button>here</button>.": "Was did niet de bedoeling? Beheer uw voorkeuren <button>hier</button>.",
+  "Don't have an account?": "Heb je nog geen account?",
+  "Edit": "Bewerken",
+  "Email": "E-mail",
+  "Email newsletter": "Nieuwsbrief",
+  "Email preference updated.": "E-mailvoorkeuren bijgewerkt",
+  "Email preferences": "E-mailvoorkeuren",
+  "Emails": "E-mails",
+  "Emails disabled": "E-mails zijn uitgeschakeld",
+  "Ends {{offerEndDate}}": "Eindigt op {{offerEndDate}}",
+  "Error": "Fout",
+  "Expires {{expiryDate}}": "Verloopt op {{expiryDate}}",
+  "Forever": "Voor altijd",
+  "Free Trial – Ends {{trialEnd}}": "Gratis proefperiode - Eindigt {{trialEnd}}",
+  "Get help": "Lees meer",
+  "Get in touch for help": "Neem contact op voor hulp",
+  "Get notified when someone replies to your comment": "Blijf op de hoogte als iemand op jouw reactie reageert",
+  "Give feedback on this post": "Deel je mening over dit artikel",
+  "Help! I'm not receiving emails": "Help! Ik ontvang geen e-mail",
+  "If a newsletter is flagged as spam, emails are automatically disabled for that address to make sure you no longer receive any unwanted messages.": "Als een nieuwsbrief als spam wordt gemarkeerd, worden e-mails automatisch uitgeschakeld voor dat adres om ervoor te zorgen dat je geen ongewenste berichten meer ontvangt.",
+  "If the spam complaint was accidental, or you would like to begin receiving emails again, you can resubscribe to emails by clicking the button on the previous screen.": "Als de spamklacht per ongeluk was, of als je weer e-mails wil ontvangen, kun je zich opnieuw abonneren op e-mails door op de knop op het vorige scherm te klikken.",
+  "If you cancel your subscription now, you will continue to have access until {{periodEnd}}.": "Als je je abonnement nu opzegt, heb je toegang tot {{periodEnd}}.",
+  "If you have a corporate or government email account, reach out to your IT department and ask them to allow emails to be received from {{senderEmail}}": "Als je een zakelijk of overheids e-mailaccount heeft, neem je contact op met uw IT-afdeling en vraag je hen toestemming te geven om e-mails te ontvangen van {{senderEmail}}",
```

Comment:
Heb je een zakelijk of overheids e-mailaccount? Neem contact op met de IT-afdeling en vraag toestemming om e-mails te ontvangen van {{senderEmail}}